### PR TITLE
chore: added one linux question

### PR DESCRIPTION
**Pull request by Dunking3022**
## Summary of changes

Added one Linux question.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const linuxQuiz = [
+  {
+    Question: "In Linux, which command option is used to perform a case-insensitive search when searching for text patterns?",
+    Answer: "grep -i",
+    Distractor1: "grep -c",
+    Distractor2: "grep -v",
+    Distractor3: "grep -s",
+    Explanation:
+      "grep -i command is used to perform a case-insensitive search, allowing you to find text patterns ignoring the case.",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:9]: small grammar change 

```suggestion
      "The grep -i command is used to perform a case-insensitive search, allowing you to find text patterns ignoring the case.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Dunking3022** [comment]
Done!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**